src/ddb/tests/ddb_vos_tests.c

```diff
@@ -894,15 +900,74 @@ clear_committed_table(void **state)
 
 	dvt_vos_insert_2_records_with_dtx(coh);
 
-	assert_int_equal(1, dv_clear_committed_table(coh));
+	assert_int_equal(1, dv_dtx_clear_cmt_table(coh));
+
+	dv_dtx_get_cmt_table(coh, committed_entry_handler, NULL);
+
+	committed_entry_handler_called = 0;
+	assert_int_equal(0, committed_entry_handler_called);
```

Comment:
Seems redundant since you just reset the "committed_entry_handler_called".

Reply:
The assignment was meant to be above the get_cmt_table function call. Fixed